packages/ui/components/Button/Button.en.mdx

```diff
@@ -34,6 +40,61 @@ Use regular Button without any options by default. Whenever you want to emphasiz
 </Button>
 ```
 
+## Colors
+
+The `color` property is `dark` by default. It can be changed by changing `color` prop. Possible values of property can be found in the `Sandbox` section at the bottom of the page.
+
+The icon color can be set by specifying the color name from $ UI.colors.
+
+The `variant` property is responsible for the external display of the button (affects the color of the icon and button text). The `variant` property is `outlined` by default.
+
```

Comment:
Про иконку выноси в отдельный раздел. Можно взять даже я думаю описание из `TextInput` раздел - иконки. Ещё посмотри `Tag` там есть про иконки. Тег наследуется от кнопки - поэтому у них какие-то описания будут даже одинаковыми. **Не забудь кстати в компоненте `Tag` тоже поправить описание что касается иконки, цветов.**

Reply:
`variant` тоже тогда нужно в отдельный раздел выносить. можно написать что-то похожее как в `Tag` компоненте - an alternative style

Reply:
Что касается раздела цвета

Color is `dark` by default. It can be changed by passing color name from the config colors to `color` property. Possible values of property can be found in the `Sandbox` section at the bottom of the page.